src/website/views/components/picker-group.pug

```diff
@@ -190,135 +223,174 @@ p.-text
 .example.-mb--3
   .-px--3.-pt--0.-pb--3
     each size in ['sm', 'md', 'lg', 'xl']
-      p.-text--bold= '-' + size
-      .-d--flex.-flex--wrap
-        .m-pickerGroup.-mr--2
-          each type in ['Radio1','Radio2','Radio3']
-            input.a-picker(type="radio", name='radios', id=`ig-${type}-${size}`)
-            label(for=`ig-${type}-${size}`, class=`-${size}`)
-                = type
-        .m-pickerGroup
-          each type in ['Radio1','Radio2','Radio3']
-            input.a-picker(type="radio", name='radios', id=`ig-i-${type}-${size}`)
-            label(for=`ig-i-${type}-${size}`, class=`-${size}`)
-              if size === 'sm'
-                i.a-icon.icon-atom.-xs
-              else
-                i.a-icon.icon-atom.-sm
-
+      fieldset
```

Comment:
Remove line 224 ` .-px--3.-pt--0.-pb--3` and wrap each fieldset example in a `.-p--3` padded div. This will provide sufficient spacing after removing the `-py--2` class from the legend.
```
.-p--3
  fieldset
```